app/views/subscription/add.phtml

```diff
@@ -27,7 +27,7 @@
 		<div class="form-group">
 			<label class="group-name" for="url_rss"><?= _t('sub.feed.url') ?></label>
 			<div class="group-controls">
-				<input id="url_rss" name="url_rss" type="url" required="required" autocomplete="off" class="long"/>
+				<input id="url_rss" name="url_rss" type="url" required="required" autocomplete="off" class="long" autofocus/>
```

Comment:
I assume we just want this one:
```suggestion
				<input id="url_rss" name="url_rss" type="url" required="required" autocomplete="off" class="long" autofocus>
```

But otherwise it should be something like this:
```suggestion
				<input id="url_rss" name="url_rss" type="url" required="required" autocomplete="off" class="long" autofocus="autofocus"/>
```
The combination's just weird. ;-)

Reply:
I've changed it